app/src/main/java/com/koalatea/thehollidayinn/softwareengineeringdaily/podcast/PodcastAdapter.java

```diff
@@ -17,13 +17,53 @@
 import java.util.ArrayList;
 import java.util.List;
 
-/*
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
  * Created by krh12 on 5/22/2017.
  */
 
-class PodcastAdapter extends RecyclerView.Adapter<PodcastAdapter.ViewHolder> {
-  private List<Post> posts = new ArrayList<>();
-  private PodListFragment context;
+public class PodcastAdapter extends RecyclerView.Adapter<PodcastAdapter.ViewHolder> {
```

Comment:
Looks like this file broke during merge. You might want to copy and paste from develop then reintroduce butterknife.